### DISCUSSION TITLE
嗯呢嗯呢呢？（有邦布大家庭的邦布吗？）

### DISCUSSION THREAD

**Discussion by mayh159** [share121/inter-knot#2496] [其他]
嗯呢嗯呢呢，嗯呢嗯呢，嗯呢呢嗯呢

**Comment by Dave-12138** [comment]
嗯呢嗯呢？<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/de1eec3712b380a1a6c2c3b7e34026e6_8819600320172395189.png" alt="米仔_问号" width="75" >

**Comment by MiCai27** [comment]
嗯呢嗯呢

**Comment by huiye123-Hub** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/04/28/b71f9bae78003f98a47f0d6e0765324d_7897172668253324041.png" alt="米仔_休息" width="75" >
